cloud/pkg/cloudhub/common/model/types_test.go

```diff
@@ -84,8 +85,9 @@ func TestEventToMessage(t *testing.T) {
 
 // TestMessageToEvent is function to test MessageToEvent
 func TestMessageToEvent(t *testing.T) {
-	msg := modelMessage("ID1", "PID1", time.Now().UnixNano()/1e6, "Source1", "Group1", "Operation1", "Resource1")
-	event := modelEvent("ID1", "PID1", time.Now().UnixNano()/1e6, "Source1", "Group1", "Operation1", "Resource1", nil)
+	timestamp := time.Now().UnixNano() / 1e6
+	msg := modelMessage("ID1", "PID1", timestamp, "Source1", "Group1", "Operation1", "Resource1")
+	event := modelEvent("ID1", "PID1", timestamp, "Source1", "Group1", "Operation1", "Resource1", nil)
```

Comment:
Those refactoring might be better in another PR since it is irrelevant with this topic, good as it as.

Reply:
the unit tests were failing when i ran the test via scripts. so I fixed them here.

Reply:
It's fine.